### ISSUE TITLE
Extra account with zero balance always added when importing wallet with seed and when first account's balance is non-zero (PR)

### ISSUE THREAD

**Issue by muinmomin** [MetaMask/metamask-extension#12073]
**Describe the bug**
When importing a wallet with a seed, and if the first account has a non-zero balance, there is always an extra account with a zero balance created.

**Steps to reproduce (REQUIRED)**

1. Install MetaMask extension (I'm using Edge/Chrome)
2. Import a wallet where the first account has a non-zero ETH balance
3. Click on profile in top right corner
4. See extra account at the end of the account list with a zero balance

**Expected behavior**
Step 4 above shouldn't happen. An extra account with a zero balance is being generated; instead, this account shouldn't be there.

**Browser details (please complete the following information):**
 - OS: macOS Big Sur and Windows 11
 - Browser: Microsoft Edge | 92.0.902.84 (Official build) (64-bit)
 - MetaMask Version: 10.1.0

**Additional context (Error Messages, etc.)**
Assuming most people's MetaMask wallets have one account with a non-zero balance. Having an extra account always be present (especially currently when there's no way to delete accounts), when importing from seed, can get annoying. It can also make people think their account has been compromised since they see an extra account there that they probably didn't have before. [Example of a probably related customer complaint here](https://community.metamask.io/t/keep-appearing-new-wallets-when-i-recover-with-seed-phrase/2143).

<br>

**Happy to submit a PR to fix this issue as it's bothering me too :) Let me know if there's some approval required before starting.**

**Comment by danjm** [comment]
Duplicate of https://github.com/MetaMask/metamask-extension/issues/11777
